.github/workflows/stable-snapshot-timestamp.yml

```diff
@@ -73,7 +78,7 @@ jobs:
 
   run_snapshot_timestamp_publish:
     needs: check
-    if: (github.event_name == 'schedule' && github.repository == 'sigstore/root-signing' && needs.check.outputs.ceremony_wip == 'false') || (github.event_name != 'schedule' && inputs.dry_run == 'false')  # Don't run workflow in forks on cron
+    if: (github.event_name == 'schedule' && github.repository == 'sigstore/root-signing' && needs.check.outputs.ceremony_wip == 'false') || (github.event_name != 'schedule' && inputs.dry_run == 'false') || (inputs.force_snapshot == 'true')  # Don't run workflow in forks on cron
```

Comment:
I think this needs to be wrapped in {{ }}

Reply:
It's not that the variable is a `boolean`, so it should be tested against `true`?